src/ui-kit/experimental/header/header.component.html

```diff
@@ -0,0 +1,106 @@
+<section class="usa-banner">
```

Comment:
@thomastighe How much would you hate it if I asked if you could put this in its own component?